Fix:
error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'memset' with type 'void *(void *, int, unsigned long)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'strcmp' with type 'int (const char *, const char *)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'strcpy' with type 'char *(char *, const char *)' [-Werror,-Wimplicit-function-declaration]
https://github.com/tklauser/libtar/pull/11
--- compat/module.ac.orig	2013-10-09 11:59:44.000000000 -0500
+++ compat/module.ac	2021-04-16 19:28:18.000000000 -0500
@@ -31,6 +31,8 @@
     [compat_cv_func_basename_works],
     [AC_TRY_RUN([
 #include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
 #ifdef HAVE_LIBGEN_H
 # include <libgen.h>
 #endif
@@ -85,6 +87,8 @@
     [compat_cv_func_dirname_works],
     [AC_TRY_RUN([
 #include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
 #ifdef HAVE_LIBGEN_H
 # include <libgen.h>
 #endif
@@ -157,6 +161,7 @@
     [compat_cv_func_glob_works],
     [AC_TRY_RUN([
 #include <stdio.h>
+#include <stdlib.h>
 #ifdef HAVE_GLOB_H
 # include <glob.h>
 #endif
@@ -208,6 +213,7 @@
     [compat_cv_func_makedev_three_args],
     [AC_COMPILE_IFELSE([
       AC_LANG_PROGRAM([[
+#include <stdlib.h>
 #include <sys/types.h>
 #ifdef MAJOR_IN_MKDEV
 # include <sys/mkdev.h>
@@ -248,6 +254,8 @@
     [compat_cv_func_snprintf_works],
     [AC_TRY_RUN([
 #include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
 
 typedef struct {
   int length;
